geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/view/MergedFeatureSourceView.scala

```diff
@@ -117,35 +112,6 @@ class MergedFeatureSourceView(
       iter
     }
 
-    override def accepts(visitor: FeatureVisitor, progress: ProgressListener): Unit =
-      visitor match {
-        case v: BoundsVisitor =>
-          v.reset(ds.stats.getBounds(sft, query.getFilter, exact = false))
-
-        case v: MinVisitor if v.getExpression.isInstanceOf[PropertyName] =>
-          val attribute = v.getExpression.asInstanceOf[PropertyName].getPropertyName
-          minMax(attribute, exact = false).orElse(minMax(attribute, exact = true)) match {
-            case Some((min, _)) => v.setValue(min)
-            case None           => super.accepts(visitor, progress)
-          }
-
-        case v: MaxVisitor if v.getExpression.isInstanceOf[PropertyName] =>
-          val attribute = v.getExpression.asInstanceOf[PropertyName].getPropertyName
-          minMax(attribute, exact = false).orElse(minMax(attribute, exact = true)) match {
-            case Some((_, max)) => v.setValue(max)
-            case None           => super.accepts(visitor, progress)
-          }
-
-        case v: GeoMesaProcessVisitor =>
-          v.execute(MergedFeatureSourceView.this, query)
-
-        case _ =>
-          super.accepts(visitor, progress)
-      }
-
-    private def minMax(attribute: String, exact: Boolean): Option[(Any, Any)] =
```

Comment:
Is this function supposed to go away?

Reply:
it's only called in the visit method, so it's in the new trait now.